neonvm-runner/cmd/main.go

```diff
@@ -983,6 +993,10 @@ func makeKernelCmdline(cfg *Config, vmSpec *vmv1.VirtualMachineSpec, vmStatus *v
 	if cfg.appendKernelCmdline != "" {
 		cmdlineParts = append(cmdlineParts, cfg.appendKernelCmdline)
 	}
+	if vmSpec.CpuScalingMode != nil && *vmSpec.CpuScalingMode == vmv1.CpuScalingModeCpuSysfsState {
+		// if we use sysfs based scaling we need to specify the start cpus as min CPUs
+		cmdlineParts = append(cmdlineParts, fmt.Sprintf("maxcpus=%d", vmSpec.Guest.CPUs.Min.RoundedUp()))
```

Comment:
Why is this required in the kernel command line when it wasn't before?

Reply:
Because without that change all cpu would be marked as online on start, while probably we want to have minimum amount of CPUs marked as online on start?

Reply:
so it is what is done for QMP hotplugs by `cpus=minCpus`, has the same semantic

Reply:
Ahhhh I see. Could you add a comment saying this?

Reply:
it actually has the comment 🤔 I tried to rephrase it to be a more clarifying, https://github.com/neondatabase/autoscaling/pull/1111/files#diff-7eb577493ab5186c29d09f0791bee695a98db31cd3619c95e834a7924744823fR1003